pattern.py

```diff
@@ -53,7 +53,7 @@ def run():
     # Smooth the result a little
     phi = gaussian_filter(phi, sigma=2.0, mode='wrap')
 
-    plt.imshow(phi, cmap='magma')
+    plt.imshow(phi, cmap='rainbow')
```

Comment:
why don't you try ocean next time? more rainbows please.

Reply:
Love this! Bright and colourful and full of joy!

Reply:
Asking questions to yourself seems kinda sad...